index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <h3>My Logo</h3>
+            </div>
+            <nav>
+                <ul>
+                 <li><a href="#">Portfolio</a></li> 
+                 <li><a href="#">About</a></li>
+                 <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+
+        </div>
+   </header>
+
+   <section class="showcase">
+       <div class="image-1">
+           <img src="./images/ellipse11.svg" alt="styling image">
+           <img src="./images/ellipse7.svg" alt="styling image">
+           <img src="./images/ellipse3.svg" alt="styling image">
+       </div>
+       <div class="hero">
+       <h1>I'm Shubh</h1>
+       <h1>Glad to see you!</h1>
```

Comment:
- [ ] Please, consider that you should use `h1` one time in your code. All other titles should be `hx` (aka h2, h3, h4...)